doc/classes/OS.xml

```diff
@@ -639,6 +639,17 @@
 				[b]Note:[/b] This method is implemented on Android, iOS, Web, Linux, macOS and Windows.
 			</description>
 		</method>
+		<method name="shell_show_in_file_manager">
+			<return type="int" enum="Error" />
+			<param index="0" name="file_or_dir_path" type="String" />
+			<param index="1" name="open_folder" type="bool" default="true" />
+			<description>
+				Requests the OS to open explorer (file manager), then navigate to the given [param file_or_dir_path] and select the target file or folder.
+				If [param file_or_dir_path] is a valid directory path, and [param open_folder] is [code]true[/code], thie method will open explorer and enter the target folder without selecting anything.
+				Use [method ProjectSettings.globalize_path] to convert a [code]res://[/code] or [code]user://[/code] path into a system path for use with this method.
+				[b]Note:[/b] Currently this method is only implemented on Windows, in other platforms, it will fallback to [method shell_open] with a directory path for [param file_or_dir_path]
```

Comment:
```suggestion
				[b]Note:[/b] Currently this method is only implemented on Windows. On other platforms it will fallback to [method shell_open] with a directory path for [param file_or_dir_path].
```